_includes/releases/v22.1/v22.1.0-alpha.4.md

```diff
@@ -0,0 +1,97 @@
+
+<h3 id="v22-1-0-alpha-4-sql-language-changes">SQL language changes</h3>
+
+- Added a `sql.auth.resolve_membership_single_scan.enabled` [cluster setting](../v22.1/cluster-settings.html), which changes the query for an internal role membership cache. Previously the code would recursively look up each role in the membership hierarchy, leading to multiple queries. With the setting on, it uses a single query. The setting is `true` by default. [#77359][#77359]
+- The [data type](../v22.1/data-types.html) of shard columns created for [hash-sharded indexes](../v22.1/hash-sharded-indexes.html) has changed from `INT4` to `INT8`. This should have no effect on behavior or performance. [#76930][#76930]
+- Introduced the `sql.contention.resolver.queue_size` metric. This gauge metric gives the current length of the queue of contention events, each awaiting translation of its transaction ID into a transaction fingerprint ID. This metric can be used to assess the level of backlog unresolved contention events. [#77514][#77514]
+- Introduced the `sql.contention.resolver.retries` metric. This counter metric reflects the number of retries performed by the contention event store attempting to translate the transaction ID of the contention event into transaction fingerprint ID. Any spike in this metric could indicate a possible anomaly in the transaction ID resolution protocol. [#77514][#77514]
+- Introduced the `sql.contention.resolver.failed_resolution` metric. This counter metric gives the total number of failed attempts to translate the transaction ID in the contention events into transaction fingerprint ID. A spike in this metric indicates likely severe failure in the transaction ID resolution protocol. [#77514][#77514]
+- Added support for `date_trunc(string, interval)` for compatibility with PostgreSQL. This built-in function is required to support Django 4.1. [#77508][#77508]
+- Introduced a `sql.contention.event_store.duration_threshold` [cluster setting](../v22.1/cluster-settings.html). This cluster setting specifies the minimum contention duration to cause the contention events to be collected into the `crdb_internal.transaction_contention_events` virtual table (default: `0`). [#77623][#77623]
+- Added support for super region functionality. Super regions allow the user to define a set of regions on the database such that any `REGIONAL BY TABLE` based in the super region or any `REGIONAL BY ROW` partition in the super region will have all their replicas in regions within the super region. The primary use is for data domiciling. Super regions are an experimental feature and are gated behind the session variable: `enable_super_regions`. The [cluster setting](../v22.1/cluster-settings.html) `sql.defaults.super_regions.enabled` is used to enable super regions (default: `false`). [#76620][#76620]
```

Comment:
Do we want to link https://www.cockroachlabs.com/docs/dev/data-domiciling.html for Data Domiciling? And maybe the Experimental Features page, don't think that is as important